lib/travis/api/v3/access_control/legacy_token.rb

```diff
@@ -8,7 +8,7 @@ class AccessControl::LegacyToken < AccessControl::User
     auth_type('token', 'basic')
 
     def self.for_request(type, payload, env)
-      payload = paylad.first if payload.is_a? Array
+      payload = payload.first if payload.is_a? Array
```

Comment:
This is a code change. I don't understand how the existing code could do anything other than crash if `payload.is_a? Array`